scipy/stats/contingency.py

```diff
@@ -271,3 +272,122 @@ def chi2_contingency(observed, correction=True, lambda_=None):
                                    lambda_=lambda_)
 
     return chi2, p, dof, expected
+
+def association(arr, chi2_stat, stat="cramer"):
+    """Calculates degree of association between variables
+    that are nominal or greater.
+
+    Allows for specification of one of four related methods,
+    Tschuprow's T, Pearson's Contingency Coefficient, Cramer's V and Phi.
+
+    Parameters
+    ----------
+    arr : array-like
+        The array of observed values
+    chi2_stat : float
+        The chi-squared statistic.
+    stat : {"cramer", "tschuprow", "pearson", "phi"} (default = "cramer")
```

Comment:
Let's eliminate the `phi` option. According to the references that I've read, Phi is only used for 2x2 arrays.  Since  Cramer's formula gives the same result as Phi in the 2x2 case, we should eliminate the Phi option, and state in the docstring that to compute the Phi coefficient for a 2x2 table, use the (default) method 'cramer'.

Reply:
I agree it was a more a function of opportunity that a real value add. I dropped it.